src/sql/src/session/vars.rs

```diff
@@ -2871,52 +2871,16 @@ impl Value for Vec<Ident> {
     }
 }
 
-impl Value for Option<String> {
+// Implement `Value` for `Option<V>` for any owned `V`.
+impl<V: Value + Clone + ToOwned<Owned = V>> Value for Option<V> {
```

Comment:
This isn't officially part of our style guide (not yet, at least) but I find it easier to read complicated bounds with a dedicated `where` clause:

```suggestion
impl<V> Value for Option<V>
where
  V: Value + Clone + ToOwned<Owned = V>
{
```

Reply:
yeah this looks better! We should update the style guide...

Reply:
(This looks unresolved!)

Reply:
lost in my `undo` spree